port_flatpak.patch

```diff
@@ -0,0 +1,166 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index 9a7cc05..0b9f93d 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -2,10 +2,15 @@
+ name = Typing Turtle
+ activity_version = 31
+ license = GPLv3+
+-bundle_id = org.laptop.community.TypingTurtle
++release_date = 2019-12-28
++bundle_id = org.sugarlabs.TypingTurtle
+ icon = Activity-typingturtle
+ exec = sugar-activity3 typingturtle.TypingTurtle
+ summary = Need some help typing? Here you will learn the best way to hold your hands in order for you to become a faster typist!
++description = Typing Turtle is an interactive touch typing program. It gradually introduces the keys on the keyboard through a series of lessons, until the student has learned the entire keyboard.
++metadata_license = CC0-1.0
++update_contact = tch@sugarlabs.org
++developer_name = Sugar Labs Community
+ repository = https://github.com/sugarlabs/typing-turtle-activity
+-tags = Language;System;Game
++tags = Education
+ url = https://help.sugarlabs.org/en/typing_turtle.html
+diff --git a/balloongame.py b/balloongame.py
+index 746b602..4c1558b 100644
+--- a/balloongame.py
++++ b/balloongame.py
+@@ -245,7 +245,7 @@ class BalloonGame(Gtk.VBox):
+     
+         cr.set_source_rgb(0, 0, 0)
+         pango_layout = PangoCairo.create_layout(cr)
+-        fd = Pango.FontDescription('Times')
++        fd = Pango.FontDescription('Monospace')
```

Comment:
What is the reason for this? Keep in mind that you an provide the font in the manifest.

Reply:
As Monospace is easily available on all systems, I chose Monospace. Times doesn't render, and I have not checked the remote repository (as I am also procastinating to do it), Basically it doesn't make much difference and we are keeping the flatpak light :)

Reply:
I am marking this as temporarily resolved :/